### PR TITLE
fix: [AA-1044] add missing h2 for screenreaders

### PR DESCRIPTION
**Pull request by cdeery**
fix: [AA-1044] add missing h2 for screenreaders

- Add placeholder h2 tag with message indicating reserve for future use

### CODE REVIEW

**Comment by cdeery** [comment]
@ciduarte Do I need to internationalize this?

File: src/courseware/course/sequence/Unit.jsx [hunk 1]
```diff
@@ -17,7 +17,7 @@ import BookmarkButton from '../bookmark/BookmarkButton';
 import { useModel } from '../../../generic/model-store';
 import PageLoading from '../../../generic/PageLoading';
 import { processEvent } from '../../../course-home/data/thunks';
-import { fetchCourse } from '../../data/thunks';
```
Comment (ciduarte) [inline src/courseware/course/sequence/Unit.jsx:20]: Was this done on purpose?

**Review by ciduarte — APPROVED** [review]
Just one question but looks good to me!

File: src/courseware/course/sequence/Unit.jsx (hunk 1 above)
Comment (cdeery) [reply src/courseware/course/sequence/Unit.jsx:20]: Yes, the commit was blocked by the linter until it was resolved.

**Pull request merged**